signer/src/config/mod.rs

```diff
@@ -328,6 +365,12 @@ impl Settings {
             .prefix_separator("_");
 
         let mut cfg_builder = Config::builder();
+
+        cfg_builder = cfg_builder.set_default("signer.dkg_max_duration", 120)?;
+        cfg_builder = cfg_builder.set_default("signer.bitcoin_presign_request_max_duration", 30)?;
+        cfg_builder = cfg_builder.set_default("signer.signer_round_max_duration", 30)?;
+        cfg_builder = cfg_builder.set_default("signer.context_window", 500)?;
```

Comment:
Have you tried commenting them out in the file and seeing what happens? Also, I think you can mark them as optional fields in the struct and that might work as well. But then we have to deal with the default value so maybe this is clearer.